### PR TITLE
update the index.html file and favorite_things file to include pictures

### PR DESCRIPTION
**Pull request by dbuford**

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -149,32 +140,24 @@ <h3> Why Computer Science? </h3>
               around us. Fortunately, since technology continues to rapidly evovle before our very eyes,
```
Comment (honkwan) [inline portfolio/src/main/webapp/index.html]: https://screenshot.googleplex.com/gZXBosEons1 is it possible to use the same width here?

**Review by honkwan — COMMENTED** [review]
Thanks for the request Dominique! The change looks mostly good :)

Do you mind sending a before and after screenshot using go/snipit? Should help a little with the review process!

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -149,32 +142,21 @@ <h3> Why Computer Science? </h3>
...
               there is always so many new things to learn. In order to continue to build upon my current coding 
               knowledge,this summer, I have the opportunity to be apart of Google's STEP, Student Training in 
-              Engineering Program, internship. 
-          </p>
+              Engineering Program, internship. </p>
 
           <div>  </div>
 
           <h3> Learn More About Me</h3>
```
Comment (honkwan) [inline portfolio/src/main/webapp/index.html]: Seems like this doesn't fully show: https://screenshot.googleplex.com/ApEzYz9tDfY

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -119,27 +119,20 @@ <h2>Dominique's Portfolio</h2>
           </ul>  
       </nav>
 
-      <article>
-          <h3> All About Me!</h3>
+      <main>
+          <h2> About Me</h2>
           <p> Hi, my name is Dominique Buford. I am from Dallas, Texas, I'm a sophomore at Duke University, and I plan
```
Comment (honkwan) [inline portfolio/src/main/webapp/index.html]: Text here will get hidden if I scroll to the top https://screenshot.googleplex.com/LXpSYGf1Jy2

**Review by honkwan — COMMENTED** [review]
Nice Dominique, I think the horizontal UI looks better than before, do you mind changing other parts to also be horizontal? https://screenshot.googleplex.com/AzrQxUnRgKp

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (honkwan) [reply portfolio/src/main/webapp/index.html]: Are you able to `resolve conversation` from your side? Feel free to close any comments that are fixed.

**Review by honkwan — COMMENTED** [review]
Maybe move the bar on top? https://screenshot.googleplex.com/KsHnB9E5yKs

**Comment by dbuford** [comment]
Screenshot 2020-05-28 at 4 56 32 PM
Screenshot 2020-05-28 at 4 56 51 PM

before and after moving pictures to right of text

**Comment by dbuford** [comment]
Screenshot 2020-05-28 at 5 40 37 PM
Screenshot 2020-05-28 at 5 50 46 PM

Before and after adding header and stretching banner in what_i_enjoy.html page

File: portfolio/src/main/webapp/favorite_things.html [hunk 4]
```diff
@@ -8,191 +8,162 @@
...
-        body
-        {
-            background-color:white;
-        }
-        img.rounded-corners
-        {
+        body{
+            background-color:White;
+            margin:0;
+            }
```
Comment (andrewhxiao) [inline portfolio/src/main/webapp/favorite_things.html:21]: minor comment, I noticed that the end braces are inconsistent: sometimes they are at the same indentation as the open brace, sometimes they are more indented. I found a handy css + html style guide here: go/htmlcssstyle. The example code shows the end brace indentation matching the start brace. Move this one back a bit. Same applies to the rest of the braces

File: portfolio/src/main/webapp/favorite_things.html [hunk 5]
```diff
@@ -8,191 +8,162 @@
...
+        /*create round images*/
+        img.rounded-corners{
             border-radius:50%;
-        }
-        /*float the image*/
+            }
+        /*style the image*/
```
Comment (andrewhxiao) [inline portfolio/src/main/webapp/favorite_things.html]: I dont see much value added by this comment. Consider removing this?

**Comment by andrewhxiao** [comment]
Overall looks great! One thing I noticed is that the navigation menu changes from white to black on some pages. https://screenshot.googleplex.com/2iiNGpM46Ai vs https://screenshot.googleplex.com/bS2WSf0FPmB.

Personally, I think having the menu remain the same for all pages makes everything feel more tied together, but I'll let you make the decision.

Also, the page background is white for all the pages, except "Hobbies and Interests" is black. I find witching from an all white page to an all black page to be a little jarring, so I would consider making all pages consistent

**Pull request merged**